save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt

```diff
@@ -154,6 +154,8 @@ class TestExecutionService(private val testExecutionRepository: TestExecutionRep
                     it.startTime = testExecDto.startTimeSeconds?.secondsToLocalDateTime()
                     it.endTime = testExecDto.endTimeSeconds?.secondsToLocalDateTime()
                     it.status = testExecDto.status
+                    it.missingWarnings = testExecDto.missingWarnings
+                    it.matchedWarnings = testExecDto.matchedWarnings
```

Comment:
Please add assert on missed/matched count in `TestExecutionControllerTest`, so that we don't run in this issue again